src/ansys/geometry/core/designer/body.py

```diff
@@ -818,6 +858,25 @@ def rotate(
             )
         )
 
+    @protect_grpc
+    @check_input_types
+    @reset_tessellation_cache
+    @min_backend_version(24, 1, 0)
```

Comment:
If that is the case (it was already available in 24R1) no need to add the decorator. We assume the first version is 24R1.
```suggestion
```